plugins/@grouparoo/klaviyo/src/lib/export/exportRecord.ts

```diff
@@ -0,0 +1,159 @@
+import { getKnownFieldMap } from "./destinationMappingOptions";
+import { Errors, ExportRecordPluginMethod } from "@grouparoo/core";
+import { connect } from "../connect";
+
+async function handleGroupsChanges(client, email, oldGroups, newGroups) {
+  const listMapByName = await client.getListMapByName();
```

Comment:
let's do some caching of this. see other plugins `listMethods` approach.

Reply:
I thought that every call for this function could affect the values there.
Let me check how the caching mechanism really works in other plugins, and how to deal with the demand of force reloading the data even if the cache key is present but the requested list ID isn't.